qtox.pro

```diff
@@ -312,6 +312,35 @@ contains(ENABLE_SYSTRAY_GTK_BACKEND, NO) {
 }
 }
 
+# The snorenotify backend implements a notification system compatible with many systems
+!android {
+contains(ENABLE_NOTIFICATION_SNORE_BACKEND, YES) {
+    DEFINES += ENABLE_NOTIFICATION_SNORE_BACKEND
+
+    unix {
+        INCLUDEPATH += "/usr/local/include/snore/core"
+        equals(QT_ARCH, x86_64) {
+            INCLUDEPATH += "/usr/local/lib64/libsnore-qt5/include"
+            INCLUDEPATH += "/usr/local/lib/x86_64-linux-gnu/libsnore-qt5/include"
+        }
+        else {
+            INCLUDEPATH += "/usr/local/lib/libsnore-qt5/include"
+            INCLUDEPATH += "/usr/local/lib/i386-linux-gnu/libsnore-qt5/include"
+        }
+
+        LIBS += -L/usr/local/lib64/ -lsnore-qt5 -lsnoresettings-qt5
```

Comment:
>…all includes refer via <libsnore/*> anyway.

Yeah, that's how it should be! Both `/usr/include` and `/usr/local/include` are default paths for linux/unix based systems. I.e. when performing the `make install` (or `make include`) targets on the lib, files will be copied to the right path depending on the architecture of the build environment. Thus, the additional include paths and the `-L…` have to be removed completely here and use the default paths to get best results for cross-platform builds.